ra/ra_test.go

```diff
@@ -667,7 +667,7 @@ func TestPerformValidationSuccess(t *testing.T) {
 
 		// Check that validated timestamp was recorded, stored, and retrieved
 		expectedValidated := fc.Now()
-		test.Assert(t, *challenge.Validated == expectedValidated, "Validated timestamp incorrect or missing")
+		test.Assert(t, expectedValidated.Equal(*challenge.Validated), "Validated timestamp incorrect or missing")
```

Comment:
IMO a better substitution here (and below) would be
```suggestion
		test.AssertEqual(t, *challenge.Validated, expectedValidated)
```